src/config/dark-sites.config

```diff
@@ -1045,18 +1048,22 @@ wtfast.com
 www.canalplus.com
 www.cc.com
 www.chiefdelphi.com
+www.deviantart.com
 www.digikam.org
 www.directvnow.com
 www.gotimelinr.com
+www.leagueoflegends.com
 www.mtv.com
 www.netflix.com
+www.patuscada.bar
```

Comment:
Not dark mode.
![image](https://user-images.githubusercontent.com/25481501/166150227-16365aa7-8713-46bd-aaf8-b56977317346.png)

Reply:
Again, why should we disable it then? Because a user going to that site expects it to be dark and will face a white site and say "WTF, I'd expect to be dark. I don't really care it's buggy, just fix it then or do a best-effort"

Reply:
https://github.com/darkreader/darkreader/pull/8807